### DISCUSSION TITLE
Timeline for updated dispute process for resolution in case of squatting

### DISCUSSION THREAD

**Discussion by trivikr** [npm/feedback#534] [Website] [answered]
Hi npm team,

The existing [Dispute Resolution](https://docs.npmjs.com/policies/disputes#when-not-to-use-this-process) documentation section states the following.
> **When not to use this process**
We are not currently accepting dispute requests to "adopt an abandoned package" or "Report Squatting" as we re-evaluate and update the overall dispute process.

Is there a timeframe for dispute process to be updated in case of "adopt an abandoned package" or "Report Squatting" status? What is the advice for current npm authors who want to publish packages under module name which is either abandoned or being squatted on?

The question I'm asking is wrt to a module whose major version was released earlier this year, and have already quadrupled in downloads. The maintainers would like to switch their ecosystem to an org (which has zero packages) which is either abandoned or being squatted on.

Regards,
Trivikram

**Comment by MylesBorins** [comment, marked as answer]
@trivikr if the org is the same name as the existing module the maintainers should open a request with support and we can see what we can do.

We are tracking the policy change work in https://github.com/npm/roadmap/issues/28 but do not have a timeline on wrapping it up.

**Reply by trivikr** [reply]
@MylesBorins The org name is different from the existing npm module.
The org name maintainers are looking for is already an existing org (either abandoned or squatted) with zero packages in it.

**Reply by MylesBorins** [reply]
@trivikr we are working on a final stretch / push of updates that we hope to roll out by the end of this quarter / early next quarter that should finally close the loop on all this. Thanks for being patient.

**Comment by trivikr** [comment]
Twitter thread explaining issue with the suspended process of transferring ownership for abandoned packages

https://twitter.com/Andrewmd5/status/1423915732979437571
